marlowe-playground-client/src/Halogen/Blockly.purs

```diff
@@ -43,12 +43,16 @@ _generator = prop (SProxy :: SProxy "generator")
 _errorMessage :: Lens' BlocklyState (Maybe String)
 _errorMessage = prop (SProxy :: SProxy "errorMessage")
 
+emptyState :: BlocklyState
+emptyState = { blocklyState: Nothing, generator: Nothing, errorMessage: Nothing }
```

Comment:
I'll bet you a croissant that `emptyState = mempty` would work.

Reply:
no can do due to a `Maybe Workspace`. https://github.com/purescript/purescript-maybe/blob/v4.0.1/src/Data/Maybe.purs#L179

Reply:
Oh. I owe you a croissant.

Reply:
I might investigate in the future if a Monoid instance of Workspace might make sense, probably not but perhaps